state/entrySyncing.go

```diff
@@ -304,10 +307,10 @@ func (s *State) GoSyncEntries() {
 				}
 			}
 
-			if s.EntryDBHeightComplete%1000 == 0 {
-				if firstMissing < 0 {
-					start = scan + 1 // If nothing is missing at scan, make sure we don't process at scan again.
-					//Only save EntryDBHeightComplete IF it's a multiple of 100 AND there are no missing entries
+			if firstMissing < 0 {
+				start = scan + 1 // If nothing is missing at scan, make sure we don't process at scan again.
```

Comment:
I think this line is supposed to be outside the modulo condition. When it's inside the condition, the dirblkSearch block is executing continuously with just a 10ms delay. When it's outside the condition it only goes in after additional blocks were saved.